src/devices/Common/System/Device/Spi/SpiBusInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Device.Spi
+{
+    public static class SpiBusInfo
+    {
+        private static int s_bufferSize = GetBufferSize();
+
+        // we would need to decide if this should throw when size can't be found or just return -1 or other placeholder
+        public static int BufferSize => s_bufferSize;
+
+        private static int GetBufferSize()
+        {
+            // non-throwing version of int.Parse(File.ReadAllText("/sys/module/spidev/parameters/bufsiz"))
+            // return -1 on error
```

Comment:
you will need to put actual implementation here 😄
- handle non-existing file
- handle badly formatted file (i.e. use int.TryParse)

Reply:
@krwq I completely misunderstood what you were asking about a pull request.  I can go ahead and make it a functional implementation.  Yes, file existence check and TryParse should be there as well as as check for RuntimeInformation.IsOSPlatform(OSPlatform.Linux) I believe.  Should I delete the previous pull request?

Reply:
you can just add a commit in this PR (push to your branch, it will auto-update)